From 5b592534be239cbc8d0964ac2bbc04323ad626a2 Mon Sep 17 00:00:00 2001
From: Qiang Yu <yuq825@gmail.com>
Date: Thu, 24 May 2018 08:36:21 +0800
Subject: [PATCH 78/83] drm/lima: add IRQF_SHARED for GP irq

On some platform all IRQs of GPU share the same line.

Signed-off-by: Qiang Yu <yuq825@gmail.com>
---
 drivers/gpu/drm/lima/lima_gp.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/lima/lima_gp.c b/drivers/gpu/drm/lima/lima_gp.c
index 280a40d..10d7ee1 100644
--- a/drivers/gpu/drm/lima/lima_gp.c
+++ b/drivers/gpu/drm/lima/lima_gp.c
@@ -224,8 +224,8 @@ int lima_gp_init(struct lima_ip *ip)
 	if (err)
 		return err;
 
-	err = devm_request_irq(dev->dev, ip->irq, lima_gp_irq_handler, 0,
-			       lima_ip_name(ip), ip);
+	err = devm_request_irq(dev->dev, ip->irq, lima_gp_irq_handler,
+			       IRQF_SHARED, lima_ip_name(ip), ip);
 	if (err) {
 		dev_err(dev->dev, "gp %s fail to request irq\n",
 			lima_ip_name(ip));
-- 
2.0.1

